### ISSUE TITLE
Error 500 when trying to export

### ISSUE THREAD

**Issue by bahuma20** [gramps-project/gramps-webapi#404]
I have a script that automatically exports GED and GRAMPS backups daily.

Since the recent update, this does not work anymore.

I get an internal server error, when doing this request:
```sh
curl --location --request POST 'https://my.host/api/exporters/gramps/file' \
--header 'Authorization: Bearer TOKEN'
```

In the logs i see this error:
```
[2023-08-03 18:58:59 +0000] [17] [ERROR] Error handling request /api/exporters/gramps/file
web_1     | Traceback (most recent call last):
web_1     |   File "/usr/local/lib/python3.9/dist-packages/gunicorn/workers/sync.py", line 135, in handle
web_1     |     self.handle_request(listener, req, client, addr)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/gunicorn/workers/sync.py", line 178, in handle_request
web_1     |     respiter = self.wsgi(environ, resp.start_response)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 2213, in __call__
web_1     |     return self.wsgi_app(environ, start_response)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 2193, in wsgi_app
web_1     |     response = self.handle_exception(e)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 2190, in wsgi_app
web_1     |     response = self.full_dispatch_request()
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1486, in full_dispatch_request
web_1     |     rv = self.handle_user_exception(e)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1484, in full_dispatch_request
web_1     |     rv = self.dispatch_request()
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1469, in dispatch_request
web_1     |     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/gramps_webapi/api/auth.py", line 44, in wrapper
web_1     |     return func(*args, **kwargs)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/views.py", line 109, in view
web_1     |     return current_app.ensure_sync(self.dispatch_request)(**kwargs)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/flask/views.py", line 190, in dispatch_request
web_1     |     return current_app.ensure_sync(meth)(**kwargs)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/webargs/core.py", line 649, in wrapper
web_1     |     return func(*args, **kwargs)
web_1     |   File "/usr/local/lib/python3.9/dist-packages/gramps_webapi/api/resources/exporters.py", line 122, in post
web_1     |     if has_permissions({PERM_EDIT_OBJ}):
web_1     | NameError: name 'PERM_EDIT_OBJ' is not defined
```

So probably something was broken in the last release

**Comment by DavidMStraub** [comment]
Oh dear. It's just a missing import, but how did that pass unit tests :scream: 

Thanks for reporting.

**Comment by DavidMStraub** [comment]
Fixed by #405.
